datasets/us/trade_csl/crawler.py

```diff
@@ -163,7 +165,10 @@ def parse_result(context: Context, result: Dict[str, Any]):
 
     for obj in parse_addresses(context, result.pop("addresses", [])):
         # h.apply_address(context, entity, obj)
-        h.copy_address(entity, obj)
+        if not entity.schema.is_a("Address"):
+            h.copy_address(entity, obj)
+        else:
+            entity.add("full", obj.get("full"))
```

Comment:
It's easier to reason about "if is an address ... else ..." than "if not is an address ... else ..."

```suggestion
        if entity.schema.is_a("Address"):
            entity.add("full", obj.get("full"))
        else:
            h.copy_address(entity, obj)
```